autogen.sh

```diff
@@ -9,7 +9,9 @@ autoreconf --install --warnings=all --force
 
 args=""
 if [ -f "$topdir/.config.args" ]; then
-	args="$args $(cat $topdir/.config.args)"
+	args="$(cat $topdir/.config.args)"
```

Comment:
That would allow putting config args files somewhere below rkt directory, so it can survive the `git clean -ffdx`.
Something like `./autogen.sh ../kvm_args` or `./autogen.sh ../coreos-no-network` or something.